src/components/pages/Contacts/index.tsx

```diff
@@ -96,11 +96,15 @@ const UnconnectedContacts: React.FC<PropsFromRedux> = ({
     setSearchQuery(event.target.value);
   };
 
+  const unauthenticated: string[] = [
+    'Expired Token',
+    'Unauthorized',
+    'Unauthenticated',
+  ];
```

Comment:
should we just export this const API utils or smt  and reuse it across the 3 pages?